CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0-beta7 (unreleased)
+
+### New Features
+- Azure Communication
```

Comment:
Should this be Azure Communication or be more specific: `Azure Communication Authentication`?

Reply:
I think "Azure Communication Common Library" would make sense.